GramAddict/core/plugin_loader.py

```diff
@@ -20,10 +23,8 @@ def __init__(self, plugin_package):
     def reload_plugins(self):
         self.plugins = []
         self.seen_paths = []
-        print()
-        print("Loading plugins . . .")
+        logger.debug("Loading plugins . . .")
```

Comment:
I think this should be info:
1. It's a subtle flex about the plugin system
2. If a new operation is added - it shows the user the new feature
3. If a user is adding their own plugin, they probably want this info up without having to throw a debug flag

Overall, it's only shown at initial script start and not too spammy. I don't think it hurts to have it be info.

Reply:
Fixed